Testscripts/Linux/CPUOfflineKernelBuild.sh

```diff
@@ -136,20 +130,35 @@ function Main() {
 			LogMsg "Did make oldconfig"
 		fi
 
-		make -j $(getconf _NPROCESSORS_ONLN)
+		if [[ $DISTRO == "redhat_8" ]] || [[ $DISTRO == "centos_8" ]]; then
+			# comment out those 2 parameters in RHEL/CentOS 8.x
+			sed -i -e "s/CONFIG_SYSTEM_TRUSTED_KEY*.*/#CONFIG_SYSTEM_TRUSTED_KEY/g" .config
+			sed -i -e "s/CONFIG_MODULE_SIG_KEY*.*/#CONFIG_MODULE_SIG_KEY/g" .config
+			sed -i -e "s/CONFIG_DEBUG_INFO_BTF*.*/#CONFIG_DEBUG_INFO_BTF/g" .config
+			yes '' | make -j $(getconf _NPROCESSORS_ONLN)
+
+		else
+			make -j $(getconf _NPROCESSORS_ONLN)
+		fi
 		LogMsg "Compiled the source codes"
 
 		make modules_install
-		LogMsg "Installed new kernel modules"
+		LogMsg "$?: Installed new kernel modules"
 
 		make install
-		LogMsg "Install new kernel"
+		LogMsg "$?: Install new kernel"
 
 		if [[ $DISTRO =~ "ubuntu" ]]; then
 			update-grub2
 			LogMsg "$?: Ran update-grub2"
 		fi
 
+		if [[ $DISTRO == "redhat_8" ]] || [[ $DISTRO == "centos_8" ]]; then
+			vmlinux_file=$(find /boot/ -name vmlinuz-5*)
```

Comment:
what's the vmlinuz-5?

Reply:
I need to pick up a new vmlinuz compressed executable file from boot directory when I run grubby command.

Reply:
if in the future, the kernel version start with 6? I think you need think about a stable way to find the new one.